onnx/defs/quantization/defs.cc

```diff
@@ -67,6 +130,45 @@ for per-tensor / per layer quantization, or a 1-D tensor for per-axis quantizati
 there's no zero point (zero point is supposed to be 0).
 )DOC";
 
+bool BuildContextDependentFunctionBodyDequantizeLinear(
+    const FunctionBodyBuildContext& ctx,
+    const OpSchema& schema,
+    FunctionProto& functionProto) {
+  FunctionBuilder builder(functionProto);
+
+  auto* input_type = ctx.getInputType(0);
+  if (!input_type->has_tensor_type()) {
+    fail_schema("Invalid x input type. Expected a tensor type");
+  }
+  auto output_elt_type = TensorProto::FLOAT;
```

Comment:
elt: is it and abbreviation?

readability: Avoid abbreviations - they add cognitive load to readers.

> Do not use abbreviations that are ambiguous or unfamiliar to readers outside the project, and do not abbreviate by deleting letters within a word.

https://google.github.io/styleguide/pyguide.html#316-naming
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
I changed to elem to match the function name. hope this works.